components/camel-ai/camel-pinecone/src/main/java/org/apache/camel/component/pinecone/PineconeVectorDbConfiguration.java

```diff
@@ -146,6 +149,17 @@ public void setTls(boolean tls) {
         this.tls = tls;
     }
 
+    public String getHost() {
+        return host;
+    }
+
+    /**
+     * Sets the API key to use for authentication
```

Comment:
Is this description correct? I would assume its some kind of hostname

Reply:
Good catch - fixed!